network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog-card.html

```diff
@@ -1,15 +1,33 @@
 {% extends "./generic-card.html" %}
 
-{% load wagtailcore_tags %}
+{% load wagtailcore_tags wagtailroutablepage_tags class_tags %}
 
-{% block tags-DISABLED-FOR-NOW %}
-<div>
-    {% with tag=page.specific.tags.first %}
-    <a class="main-tag" href="#">{{ tag }}</a>
-    {% endwith %}
+
+The following block needs to know where the blog IndexPage is,
+so it checks whether "we" are the index page, and if not, it
+crawls up the ancestor tree until it has found it.
+
+(Without it, we cannot generate the links to /blob/tag/... for
+each blog page's dominant tag)
+
+{% block tags %}
+<div class="blog-card">
```

Comment:
Let's add some spacing to the top so it looks more like the mockup. I think `mt-3` should do.

Reply:
We can't (at least not in this file), because the `blog-card` is used in multiple places (it's the same card as used in the `related blog posts` part of blog pages).